jenkins-pipelines/oss/rolling-upgrade/rolling-upgrade-gce-image.jenkinsfile

```diff
@@ -10,5 +10,5 @@ rollingUpgradePipeline(
     use_preinstalled_scylla: true,
     test_name: 'upgrade_test.UpgradeTest.test_rolling_upgrade',
     test_config: '''["test-cases/upgrades/rolling-upgrade.yaml", "configurations/gce/n2-highmem-32.yaml"]''',
-    internode_compression: 'all',
+    internode_compression: 'all'
```

Comment:
@roydahan do you think we need this one for those jobs ?

Reply:
Not necessarily.
If we have them covered in other places, we can remove here.

Reply:
ANyway, I don't think it matters

Reply:
I would argue, those jobs should use default configuration, with no extra feature.
lets remove it.